src/content/docs/browser/new-relic-browser/page-load-timing-resources/pageviewtiming-async-or-dynamic-page-details.mdx

```diff
@@ -120,9 +118,9 @@ The `BrowserInteraction` and `PageView` events end their reporting when they rec
       </td>
 
       <td>
-        [Cumulative layout shift (CLS)](https://web.dev/cls/) is available with [agent v1177 or higher](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1177). CLS is an important, user-centric metric for measuring [visual stability](https://web.dev/user-centric-performance-metrics/#types-of-metrics) because it helps quantify how often users experience unexpected layout shifts. A low CLS helps ensure that the page is [delightful](https://web.dev/user-centric-performance-metrics/#questions). This is one of three metrics identified by Google as the [Core Web Vitals](https://web.dev/vitals/).
+        [Cumulative layout shift (CLS)](https://web.dev/cls/) is available with [agent v1177 or higher](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1177). CLS is an important, user-centric metric for measuring [visual stability](https://web.dev/user-centric-performance-metrics/#types-of-metrics) because it helps quantify how often users experience unexpected layout shifts. A low CLS helps ensure that the page is [delightful](https://web.dev/user-centric-performance-metrics/#questions). This is one of three metrics identified by Google as the [core web vitals](https://web.dev/vitals/).
```

Comment:
```suggestion
        [Cumulative layout shift (CLS)](https://web.dev/cls/) is available with [agent v1177 or higher](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1177). CLS is an important, user-centric metric for measuring [visual stability](https://web.dev/user-centric-performance-metrics/#types-of-metrics) because it helps quantify how often users experience unexpected layout shifts. A low CLS helps ensure that the page is [delightful](https://web.dev/user-centric-performance-metrics/#questions).
```